engine/swarm/configs.md

```diff
@@ -612,3 +612,53 @@ configuration file.
 
 You have now updated your `nginx` service's configuration without the need to
 rebuild its image.
+
+### Example: Use templated config in Nginx service
+
+To create a configuration in which the content will be generated using a
+template engine, use the `--template-driver` parameter and specify the engine
+name as its argument. The template will be rendered when container is created.
+
+1.  Save the following into a new file `index.html`.
```

Comment:
Thinking out loud; would it be good to name this file `index.html.tmpl` so that examples below can illustrate that the filename;

- doesn't have to match the name of the config (already illustrated: the config is named `homepage`)
- doesn't have to match the name used inside the container/service